datahub-web-react/src/App.tsx

```diff
@@ -61,7 +61,19 @@ const errorLink = onError((error) => {
 const client = new ApolloClient({
     connectToDevTools: true,
     link: errorLink.concat(httpLink),
-    cache: new InMemoryCache(),
+    cache: new InMemoryCache({
+        typePolicies: {
+            Query: {
+                fields: {
+                    dataset: {
+                        merge: (oldObj, newObj) => {
+                            return { ...oldObj, ...newObj };
+                        },
```

Comment:
and going from one dataset page to another (whether that be through the UI or hitting a new link) won't result in dataset data being merged right?

Reply:
thats right - its per urn!